src/data_utils.py

```diff
@@ -207,34 +242,83 @@ def _fetch_librispeeh_txt(self) -> dict:
                 all_samples.update(samples)
         return all_samples
 
-    def _fetch_and_push_files(self, data_dir, file_paths: list, file_pattern: str):
-        """All files will be recursively collected from the `data_dir`."""
-        listdir = os.listdir(data_dir)
-        for f in listdir:
-            f = os.path.join(data_dir, f)
-            if f.endswith(file_pattern):
-                f = os.path.abspath(f)
-                file_paths.append(f)
-                continue
 
-            if os.path.isdir(f):
-                self._fetch_and_push_files(f, file_paths, file_pattern)
+class TimitDataLoader(CommonDataLoader):
+    def __init__(self, args: TimitDataLoaderArgs):
+        super().__init__(args.batch_size, args.buffer_size, args.audio_pad_id, args.labels_pad_id, args.audio_maxlen, args.labels_maxlen)
```

Comment:
Did you run the scripts through `black`. If not I would suggest doing that. It will make the linings and indentations nicer to read.